### ISSUE TITLE
[Radio] Color specs update

### ISSUE THREAD

**Issue by aweell** [dxc-technology/halstack-style-guide#302]
## Story description

- [x] Update component documentation page with corrected color specs

## React

```
color: globalTokens.black
colorOnDark: globalTokens.white
disabledColor: globalTokens.lighterBlack
disabledColorOnDark: "#575757"
disabledFontColor: globalTokens.lighterBlack
disabledFontColorOnDark: "#575757"
focusColor: globalTokens.blue
focusColorOnDark: "#1682FF"
fontColor: globalTokens.inherit
fontColorOnDark: globalTokens.white
```

## Angular **(Pending update)**

```
"--radio-color": globalTokens.black
"--radio-disabledColor": globalTokens.lighterBlack
"--radio-fontColor": globalTokens.inherit
"--radio-disabledFontColor": globalTokens.lighterBlack
"--radio-focusColor": globalTokens.blue
```

### On-light

|   Component token                 |   Element     |   Core token                      |   Value       |
| --------------------------------- | ------------- | --------------------------------- | ------------- |
|   `color`                         |   Radio       |   `Hal-Black`                    |   `#000000`   |
|   `focusColor`                    |   Radio:focus   |   `Hal-Blue-S-35`             |   `#0067b3`   |
|   `disabledColor`                 |   Radio:disabled      |  `Hal-Grey-L-75`    |   `#bfbfbf`   |
|   `fontColor`                     |   Label       |    `Hal-Black`                    |   `#000000`   |
|   `disabledFontColor`             |   Label:disabled       |   `Hal-Grey-L-75`    |   `#bfbfbf`   |

### On-dark

|   Component token                 |   Element     |   Core token                      |   Value       |
| --------------------------------- | ------------- | --------------------------------- | ------------- |
|   `colorOnDark`                   |   Radio   |   `Hal-White`                     |   `#ffffff`  |
|   `focusColorOnDark`              |   Radio:focus   |  `Hal-Blue-L-50`         |   `#0095ff`   |
|   `disabledColorOnDark`                 |   Radio:disabled      |   `Hal-Grey-S-40`   |  `#666666`  |
|   `fontColorOnDark`               |   Label       |   `Hal-White`                     |   `#ffffff`   |
|   `disabledFontColorOnDark`       |   Label:disabled      |   `Hal-Grey-S-40`   |  `#666666`   |

**Comment by mnnaveenk** [comment]
## React

color: globalTokens.black
colorOnDark: globalTokens.white
disabledColor: globalTokens.lighterBlack
disabledColorOnDark: "#575757"
disabledFontColor: globalTokens.lighterBlack
disabledFontColorOnDark: "#575757"
focusColor: globalTokens.blue
focusColorOnDark: "#1682FF"
fontColor: globalTokens.inherit
fontColorOnDark: globalTokens.white

## Angular

"--radio-color": globalTokens.black
"--radio-disabledColor": globalTokens.lighterBlack
"--radio-fontColor": globalTokens.inherit
"--radio-disabledFontColor": globalTokens.lighterBlack
"--radio-focusColor": globalTokens.blue


## Radio buttons color specs table

|   Component token                 |   Element     |   Core token                      |   Value       |
| --------------------------------- | ------------- | --------------------------------- | ------------- |
|   'color'                         |   label       |   'Hal-Black'                     |   '#000000'   |
|   'colorOnDark'                   |   container   |   'Hal-White'                     |   '#ffffff'   |
|   'disabledColor'                 |   label       |   'globalTokens.lighterBlack'     |   '#b1b1b1'   |
|   'disabledColorOnDark'           |   label       |   ''                              |   '#575757'   |
|   'disabledFontColor'             |   label       |   'globalTokens.lighterBlack'     |   '#b1b1b1'   |
|   'disabledFontColorOnDark'       |   label       |   ''                              |   '#575757'   |
|   'focusColor'                    |   container   |   'globalTokens.blue'             |   '#005FCC'   |
|   'focusColorOnDark'              |   container   |   ''                              |   '#1682FF'   |
|   'fontColor'                     |   label       |   'globalTokens.inherit'          |   'inherit'   |
|   'fontColorOnDark'               |   label       |   'Hal-White'                     |   '#ffffff'   |

**Comment by aweell** [comment]
Closed via #349

**Comment by aweell** [comment]
## [3.9.0] Updates

- [ ] Changes on the color specs of the component

**Comment by aweell** [comment]
Closed via #534
